src/js/share.js

```diff
@@ -135,6 +135,19 @@ function Share(rootEl, config) {
 		return templateUrl;
 	}
 
+	/**
+	  * Transforms the descriptive text for social links
+	  *
+	  * @private
+	  * @param {string} socialNetwork - Name of the social network that we support (e.g. twitter, facebook, linkedin, pinterest)
+	  * @returns {string}
+	  */
+	function generateDesriptiveLinkText (socialNetwork) {
+		let templateLinkText = descriptiveLinkText[socialNetwork];
+		templateLinkText = templateLinkText.replace('{{title}}', config.title);
```

Comment:
Just going to add this as some feedback that I don't feel strongly about, no right or wrong answer.

I am torn between what I think would be nicer and keeping consistency in the file.

I can understand why you have used this method to replace `{{title}}` in a string as it's the same method used to generate the `socialUrl`.

However I also think this technique makes for a more complicated file. I think it would be nicer if `descriptiveLinkText` was converted to a function that accepted the social network and the title as two parameters.

eg.

```js
function descriptiveLinkText (socialNetwork, title = '') {
    return {
        'twitter': `Share ${title} on Twitter (opens a new window)`
    }
}
```

Reply:
I did it to be consistent with the rest of the file but I understand where you're coming from...I'm happy to refactor. @JakeChampion do you have any opinions?

Reply:
Yeah the consistency argument is very valid and probably the right thing to do here. My change should probably be done as part of a wider refactor